meilisearch/index.py

```diff
@@ -1231,6 +1231,63 @@ def reset_typo_tolerance(self) -> Dict[str, Any]:
             self.__settings_url_for(self.config.paths.typo_tolerance),
         )
 
+    def get_pagination_settings(self) -> Dict[str, Any]:
+        """Get pagination settngs of the index.
+
+        Returns
+        -------
+        settings: dict
+            Dictionary containing the pagination settings of the index.
+
+        Raises
+        ------
+        MeiliSearchApiError
+            An error containing details about why Meilisearch can't process your request. Meilisearch error codes are described here: https://docs.meilisearch.com/errors/#meilisearch-errors
+        """
+        return self.http.get(self.__settings_url_for('pagination'))
```

Comment:
All the paths are defined in the `meilisearch/config.py` file like:
```
    class Paths():
        health = "health"
        keys = 'keys'
        version = 'version'
        ...
```
You could add this line at the end:
```
        pagination = 'pagination'
```
And use it like:
```suggestion
        return self.http.get(self.__settings_url_for(self.config.paths.pagination))
```